subsys/dfu/Kconfig

```diff
@@ -93,6 +93,23 @@ config UPDATEABLE_IMAGE_NUMBER
 	help
 	  If value is set to 2 or greater then, this enables support needed when
 	  application is combined with MCUboot multi-image boot.
+
+config BOOT_IMAGE_ACCESS_HOOKS
```

Comment:
I assume the answer is no, and you don't need to provide any explanation why what I suggest here is not possible:

To me it seems like we could expose these config options to MCUboot, and then replace the ones we currently have in the mcuboot repo (`bootloader/mcuboot/boot/zephyr/Kconfig`) with this one. That way we could also avoid duplicating the cmake code for adding the hooks file since the same cmake code could be executed by both mcuboot and the app.

Reply:
moved configuration to zephyr module specidic option as they are common to the application and the bootloader.

I would keep Mcuboot own Kconfig (as duplication) as well as it doesn't make any problem - just make this configurable from the MCUoot project's own menu as well.